### ISSUE TITLE
Data availability in fasta format

### ISSUE THREAD

**Issue by redna11** [yuuuuzhang/lncRNA#1]
Hi,
Do you provide a download link to all the fasta files used for your paper?
ex: D test − H − new, D test_H_newsOR,D test_M and so on.
They would be useful to try to reproduce your results.

Thanks

**Comment by irleader** [comment]
Yes, would you please kindly provide the datasets? We will be happy to use and cite your publication if the results can be reproduced.
